src/common/bfloat16.h

```diff
@@ -128,6 +130,18 @@ inline void bfloat16_t::cvt_float_to_bfloat16(const float *src, bfloat16_t *dst,
     }
 }
 
+inline void bfloat16_t::batch_cvt_float_to_bfloat16(const float *src, bfloat16_t *dst, int count){
+    constexpr int sizePerSplit = 1024;
```

Comment:
Maybe use template to define blockSize or use totalSize/totalSystemThreads to define the blockSize.